mpsl/doc/external_mpsl.rst

```diff
@@ -0,0 +1,65 @@
+.. _external_mpsl:
+
+Library internals
+#################
+
+Before using the Multi-Protocol Service Layer (MPSL) library, make sure you are familiar with its design.
```

Comment:
We should really spell that 'Multiprotocol Service Layer' according to our rules